### PR TITLE
Update es.html

### PR DESCRIPTION
**Pull request by javierobcn**
Added missing translations in spanish language

### CODE REVIEW

File: src/partials/language/es.html [hunk 1]
```diff
@@ -34,15 +34,21 @@
   "meta.source": "Fuente",
   "nav.title": "Navegación",
   "search.config.lang": "es",
+  "search.config.pipeline": "trimmer, stopWordFilter",
+  "search.config.separator": "[\s\-]+",
   "search.placeholder": "Búsqueda",
   "search.reset": "Limpiar",
+  "search.result.initializer": "Initializing search",
```
Comment (squidfunk) [inline src/partials/language/es.html:41]: Should we also translate this key?

File: src/partials/language/es.html [hunk 2]
```diff
@@ -34,15 +34,21 @@
   "meta.source": "Fuente",
   "nav.title": "Navegación",
   "search.config.lang": "es",
+  "search.config.pipeline": "trimmer, stopWordFilter",
```
Comment (squidfunk) [inline src/partials/language/es.html:37]: These keys are not necessary, as they're inherited from `en.html`.

**Review by squidfunk — CHANGES_REQUESTED** [review]
Thanks! Great to see the existing translations catching up. Could you address the requested changes?

File: src/partials/language/es.html (hunk 1 above)
Comment (javierobcn) [reply src/partials/language/es.html]: yes, done, thanks

File: src/partials/language/es.html (hunk 2 above)
Comment (javierobcn) [reply src/partials/language/es.html]: Done, thanks

**Review by squidfunk — APPROVED** [review]

**Pull request merged**